### ISSUE TITLE
NullPointerException on AnalyseRegion

### ISSUE THREAD

**Issue by jewome62** [IntellectualSites/PlotSquared#3683]
### Server Implementation

Paper

### Server Version

1.18.2

### Describe the bug

Hello,

I try update from PlotSquared-6.6.2 to 6.9.1
I have infinite loop with error (and system kill the server)

The error is 

```
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: java.lang.NullPointerException: Cannot invoke "com.sk89q.worldedit.world.block.BlockState.equals(Object)" because "now" is null
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: 	at PlotSquared-Bukkit-6.9.1-SNAPSHOT.jar//com.plotsquared.core.generator.HybridUtils.lambda$analyzeRegion$2(HybridUtils.java:246)
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: 	at PlotSquared-Bukkit-6.9.1-SNAPSHOT.jar//com.plotsquared.bukkit.util.task.BukkitPlotSquaredTask.runTask(BukkitPlotSquaredTask.java:46)
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: 	at PlotSquared-Bukkit-6.9.1-SNAPSHOT.jar//com.plotsquared.core.util.task.PlotSquaredTask.run(PlotSquaredTask.java:51)
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: 	at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftTask.run(CraftTask.java:101)
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: 	at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: 	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[01:11:06] [Craft Scheduler Thread - 3 - PlotSquared/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:833)
``` 

I link to the PR https://github.com/IntellectualSites/PlotSquared/pull/3618

And this link of code : https://github.com/IntellectualSites/PlotSquared/pull/3618/files#diff-ce65e3960e49410fbf66d7da1364b73e58821a4a9d7320e05d3b62e4c7083b5fR245-R246
I'm in a case of `now` is null.

I have rollback to 6.7.0 release (latest release before the merge of PR #3618) And i have not this problem (Other problems without crash)

### To Reproduce

Sorry I don't know how reproduce the problem
This is a server migrated version to new version since 1.13. So this is maybe edge case from this migration?

### Expected behaviour

cancel analyze if now is null ?

### Error log (if applicable)

https://paste.gg/p/jewome62/9f95f0ce8eb44cc59a52731f71f2b7ce

### Plot Debugpaste

https://athion.net/ISPaster/paste/view/eddee3788b2748c1be416cb72bb3d23a

### PlotSquared Version

PlotSquared 6.9.1

### Checklist

- [X] I have included a Plot debugpaste.
- [X] I am using the newest build from https://www.spigotmc.org/resources/77506/ and the issue still persists.

### Anything else?

DebugPaste is provider with version PlotSquared 6.7.0 (because with 6.9.1 the server did not boot well)

**Comment by dordsor21** [comment]
Can you post a log of the "infinite loop" as well?

**Comment by jewome62** [comment]
> Can you post a log of the "infinite loop" as well?

Hello,

I have the error into Error Log (https://paste.gg/p/jewome62/9f95f0ce8eb44cc59a52731f71f2b7ce) 
Here the full log of server (106MB)
[2022-06-15-4.log.gz](https://github.com/IntellectualSites/PlotSquared/files/8904588/2022-06-15-4.log.gz)

Thanks

**Comment by dordsor21** [comment]
The second issue (AIOOB exception) is fixed in IntellectualSites/FastAsyncWorldEdit#1798

**Comment by jewome62** [comment]
> The second issue (IOOB exception) is fixed in [IntellectualSites/FastAsyncWorldEdit#1798](https://github.com/IntellectualSites/FastAsyncWorldEdit/pull/1798)

Oh ! I have remove the world which caused this error (a recent plot test configuration).

I have change the archive on my previous reply. 
I was mistaken the good archive is named [2022-06-15-6.log.gz](https://github.com/IntellectualSites/PlotSquared/files/8904595/2022-06-15-6.log.gz)
